grails-app/services/I2b2HelperService.groovy

```diff
@@ -928,11 +947,16 @@ class I2b2HelperService {
             }
 
             //pad all the empty values for this column
-            for (ExportRowNew row : tablein.getRows()) {
-                if (!row.containsColumn(columnid)) {
-                    row.put(columnid, "N");
-                }
-            }
+            /* Padding empty values introduces confusion with the user.
+             * It is not really clear anymore whether there was no observation done or whether "N" has been observed.
+             * Leaving a grid cell empty when no observation exists, is a more explicit representation of the status.
+             */
+            /* for (ExportRowNew row : tablein.getRows()) {
+             *     if (!row.containsColumn(columnid)) {
+             *         row.put(columnid, "N");
+             *     }
+             * }
+             */
```

Comment:
Wouldn't it better to delete this code at all instead of commenting it. You could use above message as commit one.